optimade/models/trajectories.py

```diff
@@ -425,12 +425,14 @@ class TrajectoryResourceAttributes(EntryResourceAttributes):
         support=SupportLevel.MUST,
         queryable=SupportLevel.MUST,
     )
-    # _exmpl_hdf5file_path: Optional[str] = OptimadeField( # TODO: this field is now still visible in the retrieved data this is however not neccesary.
-    #     None,
-    #     description="""The path of the file in which the trajectory information is stored.""",  # TODO: Use pathlib for the file_path. This property probably does not need to be an OPTIMADE property because
-    #     support=SupportLevel.OPTIONAL,
-    #     queryable=SupportLevel.OPTIONAL,
-    # )
+    hdf5file_path: Optional[
+        str
+    ] = OptimadeField(  # TODO: this field is now still visible in the retrieved data this is however not neccesary.
+        None,
+        description="""The path of the file in which the trajectory information is stored.""",  # TODO: Use pathlib for the file_path. This property probably does not need to be an OPTIMADE property because
+        support=SupportLevel.OPTIONAL,
+        queryable=SupportLevel.OPTIONAL,
+    )
```

Comment:
Do this and `storage_location` really need to be exposed in the API? They seem to me like server-side parameters

Reply:
No, They do not need to exposed in the API. I already wrote this in the TODO's

Reply:
ah sorry, i missed that